script.js

```diff
@@ -1,4 +1,343 @@
+//Game Rules:
+//This game has a few modes:
+//Variables that players can adjust: Number of dice, number of players, win condition, playmode
+//Mode 1A: Highest number concatenated
+//Mode 1B: Highest number added together
+//Mode 1C: Highest number multipled
+//Mode 2A: Lowest number concatenated
+//Mode 2B: Lowest number added together
+//Mode 2C: Lowest number multipled
+
+//Game Settings input by Player
+var numberOfPlayers = ``;
+var numberOfDice = ``;
+var numberOfRounds = ``;
+var numString = [`zero`, `one`, `two`, `three`, `four`, `five`];
+
+//Playing Process
+var playersNames = [];
+var playersDiceRolls = [];
+var playersSubmissions = [];
+
+// Win-Loss Records
+var winLossRecords = [];
+var checkTiedRolls = [];
+var checkTiedWins = [];
+var tiedWinnersTemp = [];
+
+//Initial Game State
+var gameState = `summonTheGamemaster`;
+console.log(`gameState`, gameState);
+var roundsPlayed = 0;
+
+// Main Function
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  input = input.toLowerCase();
+
+  if (numString.includes(input) == true) {
+    input = numString.indexOf(input);
+  }
+
+  //Game settings code begins here
+
+  if (
+    gameState == `summonTheGamemaster` &&
+    input != `open the chamber of secrets`
+  ) {
+    return `The Gamemaster is quite a particular character. Say the words he wants or he will not entertain you.`;
+  }
```

Comment:
HAHA such a character this gamemaster. But good use of if and early return to control the flow of your code